src/wp-includes/revision.php

```diff
@@ -516,6 +516,59 @@ function wp_get_post_revisions( $post_id = 0, $args = null ) {
 	return $revisions;
 }
 
+/**
+ * Returns last revisions of specified post and count of total revisions.
+ *
+ * @since 6.1.0
+ *
+ * @param int|WP_Post|null $post Optional. Post ID or post object. `null`, `false`, `0` and other PHP falsey values
+ *                               return the current global post inside the loop. A numerically valid post ID that
+ *                               points to a non-existent post returns `null`. Defaults to global $post.
```

Comment:
```suggestion
 * @param int|WP_Post|null $post Optional. Post ID or WP_Post object. Default is global $post.
```

This matches the description of most other parameters that do the same thing.